### PR TITLE
Change log update for 10.8 release.

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by sugaarK** [comment]
I think something about the reworking of the rc smoothing to suit the new high speed links is a worthy mention and the motor mapping/direction stuff in the configurator.. remember this is likely the only thing our target audience is gona read 😂

File: changelog.html [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+            </li>
+            <li>
+                <b>Linear and Dynamic mixer options</b> - These are alternatives to the stock Betaflight mixer code. The dynamic option may result in less aggressive bump and landing responses for level mode or cinematic flights.
+            </li>
+            <li>
+                <b>So much more...</b> - The above are merely some of the enhancements and improvements. It has been awhile so 4.3 as a release is huge so make sure you check oout the complete release notes at <a href="https://betaflight.com">betaflight.com</a>.
```
Comment (limonspb) [inline changelog.html:32]: "Make sure you check oout" :)

**Review by limonspb — CHANGES_REQUESTED** [review]

**Comment by Asizon** [comment]
Agree with @sugaarK we need to make a better selection and memory of all the best improvements these 2 years in the Configurator.

File: changelog.html (hunk 1 above)
Comment (ctzsnooze) [reply changelog.html:32]: Also perhaps a link and recommendation to first read the 4.3 Tuning Guide in the Wiki at https://github.com/betaflight/betaflight/wiki/4.3-Tuning-Notes

**Comment by blckmn** [comment]
@McGiverGim any language updates?

File: changelog.html (hunk 1 above)
Comment (blckmn) [reply changelog.html]: done.

**Review by sugaarK — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by DusKing1 — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**